python/pyspark/ml/wrapper.py

```diff
@@ -263,7 +284,8 @@ def _fit_java(self, dataset):
 
     def _fit(self, dataset):
         java_model = self._fit_java(dataset)
-        return self._create_model(java_model)
+        model = self._create_model(java_model)
+        return self._copyValues(model)
```

Comment:
Yes, that is the assumption and it's the same on the Scala side too.  The estimators and models should both have a shared mixin that defines the common params used.  That's how it's done on the Scala side and Python should follow (once that's done, the temporary fix from here can be removed).